README.md

```diff
@@ -1,5 +1,6 @@
 [![slack](https://img.shields.io/badge/slack-argoproj-brightgreen.svg?logo=slack)](https://argoproj.github.io/community/join-slack)
 [![codecov](https://codecov.io/gh/argoproj/argo-cd/branch/master/graph/badge.svg)](https://codecov.io/gh/argoproj/argo-cd)
+[![Latest Version](https://img.shields.io/badge/agro--cd-v1.2.3-blue)](https://github.com/argoproj/argo-cd/releases/tag/v1.2.3)
```

Comment:
Can we name this `argo` instead of `agro` please?

Reply:
Sure, sry for the typo